scripts/valhalla_build_config

```diff
@@ -92,7 +92,8 @@ config = {
     'service': {
       'proxy': 'ipc:///tmp/thor'
     },
-    'max_reserved_labels_count': 1000000
+    'max_reserved_labels_count': 1000000,
+    'clear_reserved_memory': True
```

Comment:
> what is the difference between setting max_reserved_labels_count=0 and clear_reserved_memory=True ?

@genadz it will be no reallocations after push backing in the vector

Reply:
`false` should be the default